extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -1310,6 +1341,15 @@ static public function onWikiaRecentChangesBlockHandlerChangeHeaderBlockGroup( $
 				->rawParams( $threadLink )
 				->params( $titleData['wallTitleTxt'], $titleData['wallPageName'] )
 				->parse();
+
+			// VOLDEV-3: Bolden pages watched by this user
+			$user = $oChangeList->getUser();
+			$specialPageTitle = $oChangeList->getTitle();
+			if ( self::boldenFollowedLink( $user, $specialPageTitle, WallMessage::newFromTitle( $oTitle ) ) ) {
+				$headerTitle = '<strong class="mw-watched">' . $headerTitle . '</strong>';
```

Comment:
This line is duplicated in this file. What do you think about extracting it to separate method?